kymatio/scattering2d/frontend/torch_frontend.py

```diff
@@ -0,0 +1,110 @@
+__all__ = ['Scattering2DTorch']
+
+import torch
+import torch.nn as nn
+
+from .base_frontend import Scattering2DBase
+from kymatio.scattering2d.core.scattering2d import scattering2d
+from ...frontend.torch_frontend import ScatteringTorch
+
+
+class Scattering2DTorch(ScatteringTorch, Scattering2DBase):
+    def __init__(self, J, shape, L=8, max_order=2, pre_pad=False,
+    backend='torch'):
+        ScatteringTorch.__init__(self)
+        Scattering2DBase.__init__(self, J, shape, L, max_order, pre_pad, backend)
```

Comment:
Is there a way to pass in all the current params in fewer words?

Reply:
I don't think so.(not sure though) @eickenberg @janden @lostanlen ?